### PR TITLE
qa_crowbarsetup: add hybrid/ldap test

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by jdsn** [comment]
:+1:

**Comment by MaximilianMeister** [comment]
:+1 for code (have you tested the connection to ldap?)

**Comment by bmwiedemann** [comment]
we tested it by manually pasting snippets during development
and are doing a full fresh run with it atm

**Comment by bmwiedemann** [comment]
Looks good. After the want_ldap=1 mkcloud finished, I can do on the dashboard node
```
keystone user-get bwiedemann
keystone user-get 82608
```
to get details

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -1665,6 +1665,24 @@ function custom_configuration()
             if [[ $hacloud = 1 ]] ; then
                 proposal_set_value keystone default "['deployment']['keystone']['elements']['keystone-server']" "['cluster:$clusternameservices']"
             fi
+            if [[ $want_ldap ]] ; then
```
Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: This block made an already huge function even longer, which really impacts legibility and maintainability.  It's already too late for a -1, but any chance you could submit a new PR to extract it into a separate function?

Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: i think when we add crowbar_batch support soon, this function will not be needed anymore.

**Pull request merged**